docs/terms/validation_result.md

```diff
@@ -1,15 +1,91 @@
+
+## Use cases
+
+<CreateHeader/>
+
+A Validation Result is generated when you use [the interactive workflow to create Expectations](../guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md) and the newly created Expectation is run against the sample <TechnicalTag relative="../" tag="batch" text="Batch" /> of data you are working with.  These individual Validation Results will be created as an `ExpectationValidationResult` instance.
+
+<ValidateHeader/>
+
+Validation Results are generated when you run Checkpoints to Validate data.  Because Checkpoints operate on Expectation Suites rather than individual Expectations, the Validation Results generated by Checkpoints will be created as `ExpectationSuiteValidationResult` instances. The difference between an `ExpectationValidationResult` instance and an `ExpectationSuiteValidationResult` instance is that the `ExpectationSuiteValidationResult` will contain additional information pertaining to the Expectation Suite that was Validated, and will contain the data of an `ExpectationValidationResult` for each Expectation in the Expectation Suite.  These `ExpectationValidationResult` instances will be in a list accessed by the `results` attribute of the `ExpectationSuiteValidationResult` instance.
+
+The Validation Results generated as `ExpectationSuiteValidationResult` instances can be saved in JSON format in a Validation Result Store, and rendered as <TechnicalTag relative="../" tag="data_docs" text="Data Docs" />.  Both of these activities are handled by Actions in the Checkpoint's `action_list`.  In most cases, when you see "Validation Results" in Great Expectations, it will be in reference to the output generated as an `ExpectationSuiteValidationResult` when a Checkpoint is run.
+
+## Features
+
+### Trends in data quality
+
+Saved Validation Results allow you to track trends in how well your data adheres to Expectations.  This can help you determine if failed Expectations are due to outliers in new data, or a more systemic problem.
+
+### Actions and documentation
+
+Because Validation Results are passed to Actions in a Checkpoint's `action_list` you can create Actions that have behaviour conditional to the Validation Results.  Examples of this might be sending a Slack or email notification if the Validation fails, only launching a secondary process if the Validation Results report that Validation passed, or updating your Data Docs with specific information pulled from the Validation Results.
+
+## API basics
+
+### How to access
+
+Inside of Actions, Validation Results are a parameter that is passed to the Action when it is run.  The other place where you can access your Validation Results is in your Validation Result Store.  In both of these cases, you will be working with either an `ExpectationSuiteValidationResult` instance, or the serialization of such an instance into a JSON file.
```

Comment:
The third option here is for the case where a user might be using a Database Store Backend. I think it's a less common use case, and would add some complexity to this definition, so we should just remove the "either" here.

```suggestion
Inside of Actions, Validation Results are a parameter that is passed to the Action when it is run.  The other place where you can access your Validation Results is in your Validation Result Store.  In both of these cases, you will be working with an `ExpectationSuiteValidationResult` instance, or the serialization of such an instance into a JSON file.
```